README.md

```diff
@@ -1222,30 +1263,33 @@ The `payment` objects include additional transactional metadata:
 
 Places an order to exchange currencies.
 
+<!-- <div class='multicode'> -->
 
+*REST*
 
-```js
+```
```

Comment:
Yeah, but in the Dev Portal, the "js" gets displayed literally when it parses the code tabs, so I intentionally don't use it on the ones that are inside the multicode areas.